### PR TITLE
Feature/document/create

### PR DESCRIPTION
**Pull request by FieldPs**
### Type of change

- [x] New features
- [ ] Bug fixes
- [ ] Breaking changes
- [ ] Refactor
- [ ] Documentation updates
- [ ] Other (please specify):

## Description

* POST method to create document

### Create Document
image
image
image

### Create with missing Field
image

### Create with invalid DocType
image

### No user in Database
image
image

### CODE REVIEW

File: utils/document.utils.go
Comment (wiraphatys) [inline utils/document.utils.go:1]: could you rename this file to document_utils.go pls

Comment (FieldPs) [reply utils/document.utils.go:1]: done kub

**Review by wiraphatys — APPROVED** [review]
approve krub, lgtm

**Comment by bookpanda** [comment]
sorry for the late response 🙏 lgtm kub.

**Review by bookpanda — APPROVED** [review]

**Pull request merged**